src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
-package core.basesyntax;
+import java.util.Random;
 
 public class ColorSupplier {
```

Comment:
Consider adding an access modifier to the ColorSupplier class to specify its visibility clearly.